tests/test_ble_interface_core.py

```
@@ -766,6 +767,64 @@ def _temp_client_factory(_address: str, **_kwargs: object) -> SimpleNamespace:
+
+    def _temp_client_factory_without_kwargs(_address: str) -> SimpleNamespace:
+        return temp_client
+
+    monkeypatch.setattr(
+        "meshtastic.interfaces.ble.interface.BLEClient",
+        _temp_client_factory_without_kwargs,
+    )
+    monkeypatch.setattr(
+        iface._client_manager,
+        "_safe_close_client",
+        lambda client: cleanup_calls.append(client),
```

Ruff (0.15.5)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)